.github/scripts/fetch_outputs/main.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+
+	tfe "github.com/hashicorp/go-tfe"
+)
+
+type WorkflowRunnerConfiguration map[string]string
+
+// fetchOutputs reads the current state version for the specified workspace and returns the outputs
+func fetchOutputs(ctx context.Context, client *tfe.Client, organization string, workspace string) ([]*tfe.StateVersionOutput, error) {
+	ws, err := client.Workspaces.Read(ctx, organization, workspace)
+	if err != nil {
+		return nil, fmt.Errorf("failed reading workspace (%s/%s): %v", organization, workspace, err)
+	}
+
+	sv, err := client.StateVersions.ReadCurrentWithOptions(ctx, ws.ID, &tfe.StateVersionCurrentOptions{
+		Include: []tfe.StateVersionIncludeOpt{tfe.SVoutputs},
+	})
+	if err != nil {
+		return nil, fmt.Errorf("failed reading current state version (%s): %v", ws.ID, err)
+	} else if len(sv.Outputs) == 0 {
+		return nil, fmt.Errorf("state version (%s) has no available outputs", sv.ID)
+	}
+
+	return sv.Outputs, nil
+}
+
+// newRunnerConfiguration creates a workflow runner configuration
+func newRunnerConfiguration(ctx context.Context, outputs []*tfe.StateVersionOutput) (WorkflowRunnerConfiguration, error) {
+	config := make(WorkflowRunnerConfiguration)
+	for _, output := range outputs {
+		if output.Name == "tfe_token" || output.Name == "tfe_address" {
+			if val, ok := output.Value.(string); ok {
+				config[strings.ToUpper(output.Name)] = val
+			}
+		}
+	}
+
+	if _, ok := config["TFE_TOKEN"]; !ok {
+		return nil, fmt.Errorf("tfe_token output variable is not set")
+	}
+
+	if _, ok := config["TFE_ADDRESS"]; !ok {
+		return nil, fmt.Errorf("tfe_address output variable is not set")
+	}
+
+	tfeCfg := &tfe.Config{
+		Address:           config["TFE_ADDRESS"],
+		Token:             config["TFE_TOKEN"],
+		RetryServerErrors: true,
+	}
+
+	// We need to create a new tfe client that will talk to the tflocal instance
+	// since we need to fetch TFE_USER1 and TFE_USER2
+	client, err := tfe.NewClient(tfeCfg)
+	if err != nil {
+		return nil, err
+	}
+
+	oml, err := client.OrganizationMemberships.List(ctx, "hashicorp", &tfe.OrganizationMembershipListOptions{
```

Comment:
What is the significance of this org? Can this be extracted to a named const so the meaning is more apparent?